hw1/02-palindrome.html

```diff
@@ -9,17 +9,17 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
+  <body class="bg-primary">
     <main>
-      <section>
+      <section class="mt-5 w-50 bg-white mx-auto p-3 container">
         <h1>Exercise 02 - Palindrome</h1>
-        <div>
+        <div class="mt-3">
           <label>Enter a positive number:</label>
           <input type="number" />
         </div>
-        <div>
+        <div class="mt-3">
           <p>Is this a palindrome?</p>
-          <div></div>
+          <div id="output"></div>
         </div>
       </section>
     </main>
```

Comment:
Bootstrap sizing classes should be modified in order to match the screenshot completely.